javascript/big-practice/src/scripts/models/listNoteModel.js

```diff
@@ -49,14 +48,45 @@ export default class ListNoteModel {
   }
 
   /**
-   * @description function delete note in data
+   * @description function filter list trash
+   *
+   * @returns {Array} listNotes
+   */
+  filterTrashNotes() {
```

Comment:
We don't need to define this method, just one method `filterListNotes` is enough. You can paste a variable to get the data that you need.